Makefile

```diff
@@ -567,6 +566,13 @@ clean-assets:
 # Provided for backward compatibility with how people used to use the makefile
 update-dist: clean-assets assets
 
+# Executes whatever command is in the CMD environment variable. This is useful
+# when you want to test something in docker, e.g.
+#   CMD="go test github.com/getlantern/byteexec" make docker-exec
+exec:
+	@source setenv.bash && \
+	eval $$CMD
+
```

Comment:
I added this target to let us run arbitrary commands inside of docker.  Useful for running unit tests in Linux.